Misc/NEWS.d/next/Library/2023-01-04-12-58-59.gh-issue-100689.Ce0ITG.rst

```diff
@@ -0,0 +1 @@
+Fix crash in ``_elementtree.c`` by statically allocating ``PyExpat_CAPI`` capsule.
```

Comment:
I'd use the user visible module name instead of the source file name; the changelog is for users, not core devs :)